packages/firebase_analytics/firebase_analytics/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -151,7 +151,6 @@
 				97C146EB1CF9000F007C117D /* Frameworks */,
 				97C146EC1CF9000F007C117D /* Resources */,
```

Comment:
Could you remove those changes from the PR? same with the workspacedata file

Reply:
I hoped that they will fix the iOS build issue which I strangely can't reproduce locally or the build error not showing up in Github actions ...

Reply:
I will remove it again